src/game_initialization/lobby_info.cpp

```diff
@@ -151,7 +150,7 @@ void lobby_info::process_gamelist(const config& data)
 		qgame["experience_modifier"] = game["experience_modifier"];
 
 		qgame["mp_countdown"] = game["countdown"];
-		if(qgame["countdown"].to_bool()) {
+		if(qgame["mp_countdown"].to_bool()) {
```

Comment:
Not sure it's within the scope of this PR, but it's very odd that we have both mp_countdown and countdown keys...

Reply:
It's an existing mismatch elsewhere too, between the client internal attributes and the MultiplayerServerWML.